scene/3d/path_3d.h

```diff
@@ -64,6 +64,8 @@ class PathFollow3D : public Node3D {
 	GDCLASS(PathFollow3D, Node3D);
 
 public:
+	void _update_transform(bool p_immediate = false);
```

Comment:
If it's a public method, it shouldn't start with an underscore. You also should probably move it below where all other public methods are.

Reply:
fixed